### ISSUE TITLE
Refactor: Move SQL Queries to a Separate File

### ISSUE THREAD

**Issue by Zish303** [manascb1344/Online-Auction-System#14]
## Description:
Currently, SQL queries are defined directly within the route files. To improve code modularity and maintainability, all SQL queries should be moved to a dedicated query file. This will separate query logic from business logic, making the code cleaner and easier to manage.

## Tasks:
- Create a dbQueries.js file to store SQL queries.
- Move all SQL queries from the existing routes/controllers to the new queries file.
- Use module.exports to export the queries and require/import them in the appropriate controllers.
- Ensure that all affected routes/controllers work properly after the change.

**Comment by Zish303** [comment]
Please 'hacktoberfest' label to this issue.

**Comment by iamtehmooramjad** [comment]
Hi, I want to work on this refactoring, Can you assign it to me?

**Comment by manascb1344** [comment]
Yeah you can do it in parts as both related issues are currently assigned to you

**Comment by iamtehmooramjad** [comment]
Yes, Sure.

**Comment by Zish303** [comment]
@manascb1344 Can I work on this?
